docs/code-examples/line_segments3d_simple.cpp

```diff
@@ -5,18 +5,21 @@
 namespace rr = rerun;
 
 int main() {
-    auto rr_stream = rr::RecordingStream("rerun_example_line_segments3d");
-    rr_stream.connect("127.0.0.1:9876").throw_on_failure();
+    auto rec = rr::RecordingStream("rerun_example_line_segments3d");
+    rec.connect("127.0.0.1:9876").throw_on_failure();
 
-    std::vector<rr::datatypes::Vec3D> points = {
-        {0.f, 0.f, 0.f},
-        {0.f, 0.f, 1.f},
-        {1.f, 0.f, 0.f},
-        {1.f, 0.f, 1.f},
-        {1.f, 1.f, 0.f},
-        {1.f, 1.f, 1.f},
-        {0.f, 1.f, 0.f},
-        {0.f, 1.f, 1.f},
-    };
-    rr_stream.log("segments", rr::LineStrips3D(points));
+    // TODO(#3202): I want to do this!
```

Comment:
Nit: we don't want to leak these kinds of TODOs into the embedded snippets in the doc-pages.

Reply:
I won't do anything about that for now; we'll cleanup during the pre-release doc pass.

#3224 is there to make sure we don't forget!